#12

DIukhymenko opened the pull request:
Close #12 
Move your classes to packages.

DIukhymenko committed:
Project refactored according to standard maven project parameters.
All source files moved to src\main\java path

DIukhymenko committed:
Project divided to different packages to add logical grouping for the project needs

extsoft reviewed (changes requested):
@DIukhymenko according to https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html, there has to be a "root" package which is a reversed name of the company. Please define some company name and make proper packages structure.

DIukhymenko committed:
Refactor packages:
- add domain package
- add company name package
- add 'area' package for different types of tested area (room, flat, house, etc.)
